twisted/internet/udp.py

```diff
@@ -323,7 +323,7 @@ def writeSequence(self, seq, addr):
         @param addr: A tuple of (I{stringified IPv4 or IPv6 address},
             I{integer port number}); can be L{None} in connected mode.
         """
-        self.write("".join(seq), addr)
+        self.write(b"".join(seq), addr)
 
```

Comment:
This looks like a functional change, and most likely a good one that would fix this method on Python 3. But it doesn't have test coverage, could you add a test case for it? Do you think it would be good to split it into a separate PR?